### PR TITLE
Travis on 8.2

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
 		<admin>https://github.com/owncloud/mail#readme</admin>
 	</documentation>
 	<dependencies>
-		<owncloud min-version="7.0" max-version="8.2" />
+		<owncloud min-version="7.0" max-version="8.3" />
```
Comment (ChristophWurst) [inline appinfo/info.xml]: shouldn't max-version be changed to 9.0 like current core master?

Comment (DeepDiver1975) [inline appinfo/info.xml]: hmmm .... I did not test this yet? :speak_no_evil:

**Comment by DeepDiver1975** [comment]
9.0 is now max req and travis passes - ready to merge

@Gomez @ChristophWurst THX

**Comment by Gomez** [comment]
:+1:

**Pull request merged**